pages/catalog/CatalogHandler.inc.php

```diff
@@ -113,7 +183,7 @@ function category($args, $request) {
 
 			// Expose the featured monograph IDs and associated params
 			$featureDao = DAORegistry::getDAO('FeatureDAO');
-			$featuredMonographIds = $featureDao->getSequencesByAssoc(ASSOC_TYPE_CATEGORY, $category->getId());
+			$featuredMonographIds = $featureDao->getMonographIdsByAssoc(ASSOC_TYPE_CATEGORY, $category->getId());
```

Comment:
The schema uses `submission_id` rather than `monograph_id`, so I'd suggest doing the same in the function calls -- i.e. `getSubmissionIdsByAssoc`.

Reply:
I didn't introduce `FeatureDAO::getMonographIdsByAssoc()`. Are you suggesting I modify it, and change the name everywhere it's used now? Should I do the same for the same method in the `NewReleaseDAO`?

Reply:
Hrm, you're right, it's already there. Leave it for now; it should be standardized, but let's do it outside of this change.